pkg/apis/kops/cluster.go

```diff
@@ -220,6 +220,9 @@ type AWSSpec struct {
 	// Spotinst cloud-config specs
 	SpotinstProduct     *string `json:"spotinstProduct,omitempty"`
 	SpotinstOrientation *string `json:"spotinstOrientation,omitempty"`
+
+	// BinaryLocation is the location of the AWS cloud provider binaries.
+	BinaryLocation *string `json:"binaryLocation,omitempty"`
```

Comment:
`BinaryLocation` seems a little vague to me.

Reply:
Yeah I know. I hate it. But I couldn't find a good name for this.

FTR: the motivation behind this one is to use the PR/staging bucket in the AWS cloud provider e2e tests.